packages/_site/src/pages/components/switch/index.tsx

```diff
@@ -119,6 +121,163 @@ export default function SwitchExample() {
                                 />
                             </div>
                         </div>
+                        <div>
+                            <ExampleBlock
+                                title="Medium"
+                                description="For more important actions, set size to medium."
```

Comment:
The sizing isn't related to the importance of an action. That portion of the description can be removed same goes for the other descriptions.